lte/gateway/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What's the purpose of adding this file in this PR?

Reply:
Thank you for catching this anomaly. This change was intended for a different branch.